.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
                 name: Install dependencies (dash)
                 command: |
                     git clone git@github.com:plotly/dash.git
-                    git clone git@github.com:plotly/dash-renderer.git
+                    git clone --branch chriddyp-dev-tools git@github.com:plotly/dash-renderer.git
```

Comment:
I'll remove this before merging. Just making sure that the other repo's prop checking doesn't cause these tests to fail.